apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -372,7 +379,10 @@ export const CssValueInput = ({
 
     // If the menu is open and visible we don't want to trigger onChangeComplete
     // as it will be done by Downshift
+    // There is situation that Downshift will not call omCompleted if nothing is selected in menu
     if (isOpen && !menuProps.empty) {
```

Comment:
also  would be nice :)
```suggestion
    if (isOpen && menuProps.empty === false) {
```